### DISCUSSION TITLE
Settings sections that aren't selected looks like they are disabled

### DISCUSSION THREAD

**Discussion by toodols** [ppy/osu#17519] [Ideas] [answered]
(Really minor thing)

When I want to change my screen mode from borderless to fullscreen (one has "lower input latency" and the other can't record), I search up "full" or "border". There are 2 "sections" and the bottom one looks like they're disabled. You get confused for a half a second and then realize oh wait I have to click on it. I just think it's not intuitive even though it's like a button click. I think it should either be not disabled at all (don't know the technicalities) or the whole thing should be highlighted so it indicates that it can be clicked on.

<img width="553" alt="Screen Shot 2022-03-28 at 6 16 52 PM" src="https://user-images.githubusercontent.com/67543533/160513098-6bc4a4b2-4613-4385-baa5-a11e6cf12b3b.png">

**Comment by bdach** [comment]
this was sort of intended (#14384). likely to make the settings menu less visually overwhelming, i suppose?

**Reply by peppy** [reply]
It might help to also dim the background of the "inactive" panels.

**Reply by frenzibyte** [reply]
Gave the background dim a try, and I'm slightly unsure if it feels too dimmed or looks perfect:

https://user-images.githubusercontent.com/22781491/163286936-4f8d067a-5b4c-429d-b350-089dbf202391.mp4

**Reply by peppy** [reply]
Hard to tell with the video compression - can you PR this change? I think it's probably a good direction.

**Comment by frenzibyte** [comment, marked as answer]
Will be resolved in https://github.com/ppy/osu/pull/17822.
